.github/scripts/trymerge.py

```diff
@@ -463,6 +463,50 @@ def get_check_run_name_prefix(workflow_run: Any) -> str:
     else:
         return f'{workflow_run["workflow"]["name"]} / '
 
+
+def add_workflow_conclusions(
+    checksuites: Any,
+    get_next_checkruns_page: Callable[[List[Dict[str, Dict[str, Any]]], int, Any], Any],
+    get_next_checksuites: Callable[[Any], Any]
+) -> Dict[str, Tuple[str, str]]:
+    conclusions = {}
+
+    def add_conclusions(edges: Any) -> None:
```

Comment:
May be add an unit test or two for `add_workflow_conclusions` now that it's testable?

Reply:
im disinclined to write tests for it because `get_land_checkrun_conclusions` is basically a wrapper for `add_workflow_conclusions` and i think id pretty much be writing `get_land_checkrun_conclusions` in test form

i did change some of the tests to check for more exact numbers for the conclusions length and I made sure these numbers matched with what it was before i made changes